<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SummaryChartsLeftWidget</class>
 <widget class="QWidget" name="SummaryChartsLeftWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="topWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>44</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#topWidget{
	background-color:#ffffff;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:16px;
color:#333333;</string>
        </property>
        <property name="text">
         <string>实时销售汇总</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="leftWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="lastPageButton">
              <property name="minimumSize">
               <size>
                <width>87</width>
                <height>37</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>87</width>
                <height>37</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
	border-image: url(:/saleImage/btn-up-nomal.png);
}

QPushButton:pressed
{
	border-image: url(:/saleImage/btn-up-pess.png);
}

QPushButton:disabled
{
border-image: url(:/saleImage/btn-up-disable.png);
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_3" native="true">
              <property name="minimumSize">
               <size>
                <width>1</width>
                <height>18</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>1</width>
                <height>18</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border: 1px solid #ededed;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="nextPageButton">
              <property name="minimumSize">
               <size>
                <width>87</width>
                <height>37</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>87</width>
                <height>37</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	border-image: url(:/saleImage/btn-down-nomal.png);
}

QPushButton:pressed{
	border-image: url(:/saleImage/btn-down-pess.png);
}

QPushButton:disabled{
	border-image: url(:/saleImage/btn-down-disabled.png);
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="pageLabel">
           <property name="styleSheet">
            <string notr="true">font-size:12px;
color:#999999;
font-family:&quot;微软雅黑&quot;;
padding-top:6px;</string>
           </property>
           <property name="text">
            <string>1/1页</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="SummaryChartsLeftTitleWidget" name="titleWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="SummaryChartsLeftListView" name="listView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QListView{
	border:none;
}</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bottomWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#bottomWidget{
	background-color:#ffffff;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:14px;
color:#333333;</string>
        </property>
        <property name="text">
         <string>合计</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="saleSumMoneyLabel">
        <property name="minimumSize">
         <size>
          <width>113</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>113</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:14px;
color:#ea8852;</string>
        </property>
        <property name="text">
         <string>10000000.11</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="peopleSumNumberLabel">
        <property name="minimumSize">
         <size>
          <width>127</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>127</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:14px;
color:#ea8852;</string>
        </property>
        <property name="text">
         <string>10000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SummaryChartsLeftTitleWidget</class>
   <extends>QWidget</extends>
   <header>SummaryChartsLeftTitleWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SummaryChartsLeftListView</class>
   <extends>QListView</extends>
   <header>SummaryChartsLeftListView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
